### ISSUE TITLE
Fix handling of non-ascii titles

### ISSUE THREAD

**Issue by LeadSongDog** [internetarchive/openlibrary-client#144]
Accessing 
https://openlibrary.org/books/OL3081396M/Su%CC%84gakuteki_sekaikan
or even just 
https://openlibrary.org/books/OL3081396M
generates the error message

/opt/openlibrary/deploys/openlibrary/e10c3f4/openlibrary/templates/site/neck.html: error in processing template: UnicodeDecodeError: 'ascii' codec can't decode byte 0xcc in position 2: ordinal not in range(128) (falling back to default template)

**Comment by hornc** [comment]
:( I was hoping this was fixed with https://github.com/internetarchive/openlibrary/pull/2181 clearly not. It might be a similar problem so I'll look at it. Thanks for the further example!

**Comment by hornc** [comment]
see also https://openlibrary.org/works/OL17630832W/Principios_Naturales_de_la_Mente_III?debug=true

**Comment by hornc** [comment]
I was unable to reproduce this on my local dev environment, and I believe the difference is down to a difference in versions of web.py on prodcution and in the current docker dev environment.

Production is at `web.py (0.33)` while docker dev is `web.py (0.39)`  Web.py had multiple unicode handling fixes in 0.35 : https://github.com/webpy/webpy/blob/master/ChangeLog.txt#L65  

I did not realise production was behind in its web.py version. We need to update this! @mekarpeles 

 I think combined with the change I made recently it will fix this error. It is possible my 'fix' for 0.39 has actually caused new (or different) unicode problems in production :( Matching the web.py versions shoudl fix things, or at least show us what problems are remaining that need further attention.
